### PR TITLE
Sjapril2020

### PR DESCRIPTION
**Pull request by sjamshahi**

### CODE REVIEW

File: distributions/Dist_sj2879.py [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
 import numpy as np
 from .base_distribution import BaseDistribution
 
 class Dist_sj2879(BaseDistribution):
 	def __init__(self):
-		self.f_max = 1
-		self.x_min = -1
-		self.x_max = 1
-
+		self.f_max = 36
```
Comment (cranmer) [inline distributions/Dist_sj2879.py]: This was the right value before you normalized. But now it is larger than it needs to be. Your normalized PDF is x**2/18, so f_max just needs to be 3**2/18 = 0.5

**Review by cranmer — COMMENTED** [review]
This code will work, but it will be inefficient because f_max is very large.

File: distributions/Dist_sj2879.py (hunk 1 above)
Comment (sjamshahi) [reply distributions/Dist_sj2879.py]: Okay I just fixed this in my branch on the github desktop app, but for some reason, I can't send another pull request

Comment (sjamshahi) [reply distributions/Dist_sj2879.py]: I am assuming I don't need to?

**Comment by cranmer** [comment]
You shouldn’t need to send another pull request, but you do need to push
your changes to GitHub

**Comment by sjamshahi** [comment]
<!--
/* Font Definitions */
@font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;}
/* Style Definitions */
p.MsoNormal, li.MsoNormal, div.MsoNormal
	{margin:0in;
	margin-bottom:.0001pt;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
a:link, span.MsoHyperlink
	{mso-style-priority:99;
	color:blue;
	text-decoration:underline;}
.MsoChpDefault
	{mso-style-type:export-only;}
@page WordSection1
	{size:8.5in 11.0in;
	margin:1.0in 1.0in 1.0in 1.0in;}
div.WordSection1
	{page:WordSection1;}

**Pull request merged**